.travis.yml

```diff
@@ -13,3 +13,4 @@ script:
   - cmake -DCMAKE_BUILD_TYPE=Release -DDEPENDENCY_SEARCH_PREFIX=$VIRTUAL_ENV -DAUTOBUILD_TESTS=1 -DAUTOEXEC_TESTS=0 -DVIGRANUMPY_LIBRARIES="/opt/python/${TRAVIS_PYTHON_VERSION}*/lib/libpython${TRAVIS_PYTHON_VERSION}.so;/usr/lib/libboost_python.so" ..
   - make
   - make test
+  - ./test/blockwisealgorithms/test_blockwiselabeling
```

Comment:
This was to debug on travis.
Normal test suite does not show enough errors /couts.
Should I remove and commit? Or how should we handle this =)